contrib/windeploy/detached-sig-create.sh

```
@@ -1,5 +1,6 @@
+
 #!/bin/sh
```

Shellcheck (0.11.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)